spread/project.go

```diff
@@ -122,6 +122,9 @@ type System struct {
 	// Only for Linode and Google so far.
 	Storage Size
 
+	// Only for Google so far.
+	SecureBoot	bool
```

Comment:
Every other option we have with multiple words use dashes to separate them. We should do this here as well.